### PR TITLE
Feature/is53/projects schemas routers

### PR DESCRIPTION
**Pull request by sj-lt**
Think i fixed everything from the review.

### CODE REVIEW

File: routes/projects.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    console.log("Post Project start");
+
+    const name = req.body.name;
+    const usernameId = req.body.userId;
+    let valid = 'true';
+    if (typeof usernameId === 'undefined') {
+        valid = 'false'
+    }
+    if (valid === 'true') {
+        let response = projectTemplate_;
```
Comment (kulaszewicz) [inline routes/projects.js]: const is here is better, u don't reassign it to anything just modify properties inside.

File: routes/projects.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        response["userId"] = usernameId;
+        response["name"] = name;
+        //TODO save data
+        res.status(200).send(response);
+    }
+    else {
+        console.log("no argument");
+        let response = {
```
Comment (kulaszewicz) [inline routes/projects.js]: same here, const is good as well

**Review by kulaszewicz — APPROVED** [review]
Overall everything seems fine, small parts can be changed but I approve it.

**Review by Rustyiller — APPROVED** [review]
i hope i fixed it and it works

**Pull request merged**